neon_tts_plugin_larynx_server/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 import requests
 from ovos_plugin_manager.templates.tts import TTS, TTSValidator
-from neon_utils.parse_utils import format_speak_tags
+# from neon_utils.parse_utils import format_speak_tags
```

Comment:
@JarbasAI I think this was ported to ovos-plugin-manager, can you confirm which version got that change?